doc/nrf/gs_testing.rst

```diff
@@ -21,3 +21,25 @@ Connect with the following settings:
  * 1 stop bit
  * No parity
  * HW flow control: None
+
+.. _lte_connect:
+
+How to connect with LTE Link Monitor
+************************************
+
+To connect to nRF9160-based boards (for example, the nRF9160 DK or Nordic Thingy:91), you can also use `LTE Link Monitor`_, which is implemented in `nRF Connect for Desktop`_.
+This application is used to establish LTE communication with the nRF9160 modem through AT commands, and it also displays the UART output.
+
+To connect to the nRF9160-based board with LTE Link Monitor, perform the following steps:
+
+1. Launch the LTE Link Monitor app.
+
+   .. note::
+
+      Make sure that **Automatic requests** is enabled in LTE Link Monitor.
+
+#. Connect the nRF9160-based board to the PC with a USB cable.
+#. Power on the nRF9160-based board.
+#. Click **Select Device** and select the particular board entry from the drop-down list in the LTE Link Monitor.
+#. Observe that the LTE Link monitor app starts AT communication with the modem of the nRF9160-based board and shows the status of the communication in the display terminal.
```

Comment:
Thingy:91 does start the application firmware when powering on, but the DK doesn't (at least my 0.7.0 DK doesn't). So it would make sense to mention that in case of the DK, either before or after _select device_ the reset button must be pressed on the DK to start the application (e.g. AT_client).

Reply:
Adding the following as a note after Select Device and after the initial display of communication: 
In the case of nRF9160 DK, the reset button must be pressed to restart the device and to start the application.